Libraries/Share/Share.js

```diff
@@ -58,8 +58,8 @@ class Share {
    */
   static share(content: Content, options: Options = {}): Promise<Object> {
     invariant(
-      typeof content === 'object' && content !== null,
-      'Content must a valid object'
+      typeof content === 'object' && content != null,
+      'Content must be a valid share content object'
```

Comment:
What is a "share content object"?

This sounds sufficient to me:

> Content must be a valid object

Thanks for the fix!